services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/IslCostSpec.groovy

```diff
@@ -131,4 +131,45 @@ class IslCostSpec extends BaseSpecification {
             assert islUtils.getIslInfo(links, reverseIsl).get().state == IslChangeType.DISCOVERED
         }
     }
+
+    def "ISL(not BFD) is NOT FAILED earlier than discoveryTimeout is exceeded when connection is lost(not port down)"() {
+        given: "ISL going through a-switch and state of ISL"
```

Comment:
what means `and state of ISL` ?

Reply:
I mean we have status of Isl which is going through a-switch

Reply:
I find this info redundant, but you can leave it if you want to